### ISSUE TITLE
TODO: Resolve the edge case where two people push the same nextTag at the same time

### ISSUE THREAD

**Issue by alex-astronomer** [astro-projects/astro-cli#9]
Found in cloud/deploy/deploy.go.

https://github.com/astro-projects/astro-cli/blob/f5f8330289de0280c0596262f040bfbcff075ad0/cloud/deploy/deploy.go#L128

Does this problem still exist?

**Comment by kushalmalani** [comment]
Yes it does. its a minor one and we havent had issues with this one yet.

Going to update the description from an open ticket and we can track this ticket here.

**Comment by kushalmalani** [comment]
Migrating this ticket to astro-cli repo. New ticket is linked
